### ISSUE TITLE
Get rid of snip-snip

### ISSUE THREAD

**Issue by dschwen** [idaholab/moose#9069]
### Description of the enhancement or error report
Could we please get rid of the
```
**START JSON DATA**
```

```
**END JSON DATA**
```

Snip marks? 

### Rationale for the enhancement or information for reproducing the error
There is no output before or after these markers anyways, so they are not helpful but just a minor annoyance.

### Identified impact
Makes parsing the JSON slightly easier.

**Comment by permcody** [comment]
Well first thing you do is copy the file to your home directory. Then you edit it. Then you open Peacock... 😆

**Comment by brianmoose** [comment]
Well, apps tend to output there banner stuff before everything. For example, BISON has its big NOTICE and EXPORT RESTRICTIONS printed out before the JSON.

**Comment by andrsd** [comment]
If you patch `MooseApp` so that it does not print header in `setupOptions()` when doing JSON and other dumps, you should be able to achieve this easily.

**Comment by brianmoose** [comment]
Even with that, there is still a chance there will be output besides the JSON.
For example, if you are using PETSc debug, it prints some stuff out at the end (see #9032 ).

**Comment by brianmoose** [comment]
I don't think this is feasible since we really can't guarantee that the JSON output is the only output.
